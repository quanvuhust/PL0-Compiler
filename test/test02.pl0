Program Example2; 
Var a;
Procedure gcd(b);  
    Begin    
       a := b;
    End;
Begin
    Call gcd(12);
End. 