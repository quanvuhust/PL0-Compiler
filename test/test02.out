j 6
int 5
la 1, 0
lv 0, 4
st
ep
int 5
int 4
lc 12
dct 5
call 0, 1
hlt
